hpx/plugins/parcel/coalescing_message_handler_registration.hpp

```diff
@@ -47,8 +47,8 @@ namespace hpx { namespace parcelset
 #define HPX_REGISTER_COALESCING_COUNTERS(Action, coalescing_name)             \
     namespace hpx { namespace parcelset                                       \
     {                                                                         \
-        template<>                                                            \
-        inline char const* get_action_coalescing_name<Action>()               \
+        template<> HPX_ALWAYS_EXPORT                                          \
+        char const* get_action_coalescing_name<Action>()                      \
```

Comment:
The rationale is that we don't really have to have the `get_action_coalescing_name` definition in each translation unit. We have the same already for `get_action_name`.